### PR TITLE
Enable resources

### PR DESCRIPTION
**Pull request by ccraluca**
Enable I2C, SPI, Serial Port, GPIO, and 1-Wire.

### CODE REVIEW

**Comment by SRaus** [comment]
Please put those commits in 2 separate PRs(so will have to have two dev branches), just because "Delete all default files..." is pretty straight forward and can be merged quickly, but on enabling resources we still have tests to do.
Also, please create one more PR to push "Delete all default files..." in current release branch (2021_R2).
Thanks.

**Comment by ccraluca** [comment]
So for "Delete all default files.." you mean to have two PR's? One for aster and one for 2021_R2?

**Comment by SRaus** [comment]
> So for "Delete all default files.." you mean to have two PR's? One for master and one for 2021_R2?

Yes.

File: stage7/02-enable-resources/00-run.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+
+sed 's/#dtparam=i2c_arm=on/dtparam=i2c_arm=on/g' config.txt
+sed 's/#dtparam=spi=on/dtparam=spi=on/g' config.txt
+echo "dtoverlay=rgpio=on" | tee -a config.txt
```
Comment (mirceacaprioru) [inline stage7/02-enable-resources/00-run.sh:7]: just do `echo "dtoverlay=rgpio=on" >> config.txt` this appends to a file

**Pull request closed without merging**